nbs/tutorials/01_quickdraw.py

```diff
@@ -0,0 +1,28 @@
+from fasthtml.common import *
+from datetime import datetime
+
+def render(room):
+    return Li(A(room.name, href=f"/rooms/{room.id}"))
+
+app,rt,rooms,Room = fast_app('data/drawapp.db', hdrs=(picolink,), render=render, id=int, name=str, created_at=str, pk='id')
```

Comment:
picolink is included by default.
```suggestion
app,rt,rooms,Room = fast_app('data/drawapp.db', render=render, id=int, name=str, created_at=str, pk='id')
```